frontend/src/app/recipe/components/detail/recipe-detail.component.html

```diff
@@ -1,35 +1,32 @@
 @if (_recipe$ | async; as recipe) {
-<h1>{{recipe.name}}</h1>
+  <h4>{{ recipe.name }}</h4>
 
-@if (recipe.imageUrl) {
-  <img [src]="recipe.imageUrl">
-} @else {
-  <img src="dummy-image-square-300x300.jpg">
-}
+  @if (recipe.imageUrl) {
+    <img [src]="recipe.imageUrl">
+  } @else {
+    <img src="dummy-image-square-300x300.jpg">
+  }
 
-<div class="row">
-    <div class="col-md-8">
-        <h3>Instructions:</h3>
-        <ul>
-            @for (block of recipe.instructions; track $index) {
-            <p>
-              @for(line of block; track $index){
-                <li>{{ line }}</li>
-              }
-            </p>
-            }
-        </ul>
-    </div>
+  <div class="row">
     <div class="col-md-4">
-        <h3>Ingredients:</h3>
-        <ul>
-            @for (ingredient of recipe.ingredients; track $index) {
-            <li>{{ingredient}}</li>
+      <h6>Zutaten:</h6>
```

Comment:
Lets put h2 here.

Reply:
I think h2 is to big.

Reply:
h2 is needed on the accessibility perspective: every page needs only one h1 and the next level of headlines needs to be h2 and then h3 and so on. If that is too big, we use the class h5 or h6 here: `<h2 class="h5">`